File: 1030. Matrix Cells in Distance Order.py
class Solution(object):
    def allCellsDistOrder(self, rows, cols, rCenter, cCenter):
        ans = []
        for i in range(rows):
            for j in range(cols):
                ans.append([i, j])
        ans.sort(key=lambda x: abs(rCenter - x[0]) + abs(cCenter - x[1]))
        return ans
        

File: 1030. Matrix Cells in Distance Order.php
class Solution {
    function allCellsDistOrder($rows, $cols, $rCenter, $cCenter) {
    $ans = [];
    for ($i = 0; $i < $rows; $i++) {
        for ($j = 0; $j < $cols; $j++) {
            $ans[] = [$i, $j];
        }
    }
    usort($ans, function($r, $c) use ($rCenter, $cCenter) {
        return abs($rCenter - $r[0]) + abs($cCenter - $r[1]) - abs($rCenter - $c[0]) - abs($cCenter - $c[1]);
    });
    return $ans;
    }
}
